content/applications/websites/livechat/responses.rst

```diff
@@ -160,24 +152,87 @@ end the conversation.
 Canned responses
 ================
 
-*Canned responses* are customizable inputs where a *shortcut* stands in for a longer response. An
-operator will enter the shortcut, and it will automatically be replaced by the expanded
-*substitution* response in the conversation.
+*Canned responses* are customizable inputs where a typed shortcut populates a longer response. A
+user enters a keyword shortcut, which is then automatically replaced by the expanded substitution
+response. Canned responses save time by allowing users to use shorthand phrases to populate longer
+messages. This also limits the possibility of errors when typing out longer messages because these
+are pre-set messages. This maintains consistency throughout customer interactions.
+
+Canned responses consist of two main components: the *shortcut* and the *substitution*. The shortcut
+is the keyword or key phrase that is to be replaced. The substitution is the longer message that
+replaces the shortcut.
```

Comment:
Since you kind of explain this above, how about merging this paragraph with the explanation on lines 155 - 157, so that similar information isn't repeated?